src/diffusers/loaders.py

```diff
@@ -173,6 +188,7 @@ def map_from(module, state_dict, *args, **kwargs):
 class UNet2DConditionLoadersMixin:
     text_encoder_name = TEXT_ENCODER_NAME
     unet_name = UNET_NAME
+    aux_state_dict_populated = None
```

Comment:
should we really make this a class variable? Think we could maybe just remove it here no and check with `getattr(...)` instead
 ```suggestion
```

Reply:
Done in [172f572](https://github.com/huggingface/diffusers/pull/4147/commits/172f57266daa9f850847b5854b6b91559d43366f)